packages/RNTester/js/components/RNTesterExampleList.js

```diff
@@ -72,16 +108,43 @@ class RowComponent extends React.PureComponent<{
                   styles.row,
                   {backgroundColor: theme.SystemBackgroundColor},
                 ]}>
-                <Text style={[styles.rowTitleText, {color: theme.LabelColor}]}>
-                  {item.module.title}
-                </Text>
-                <Text
-                  style={[
-                    styles.rowDetailText,
-                    {color: theme.SecondaryLabelColor},
-                  ]}>
-                  {item.module.description}
-                </Text>
+                <View style={styles.rowTextContent}>
+                  <Text
+                    style={[styles.rowTitleText, {color: theme.LabelColor}]}>
+                    {item.module.title}
+                  </Text>
+                  <View style={{flexDirection: 'row', marginBottom: 5}}>
+                    <Text style={{color: 'blue'}}>Category: </Text>
+                    <Text>{item.module.category || 'Components/Basic'}</Text>
+                  </View>
+                  <Text
+                    style={[
+                      styles.rowDetailText,
+                      {color: theme.SecondaryLabelColor},
+                    ]}>
+                    {item.module.description}
+                  </Text>
+                </View>
+                <View
+                  style={{
+                    flex: 0.15,
+                    justifyContent: 'space-between',
+                    alignItems: 'center',
+                  }}>
+                  <TouchableHighlight
+                    style={styles.imageViewStyle}
+                    onPress={this.onButtonPress}>
+                    <Image
+                      style={styles.imageStyle}
+                      source={
+                        this.state.active
+                          ? require('../assets/bookmark-filled.png')
```

Comment:
Similarly, for bookmarks, we can reuse them in the bottom tab bar

Reply:
This has been implemented.